sdcm/nemesis.py

```diff
@@ -340,6 +341,11 @@ def repair_nodetool_rebuild(self):
         rebuild_cmd = 'nodetool -h localhost rebuild'
         self._run_nodetool(rebuild_cmd, self.target_node)
 
+    def disrupt_nodetool_cleanup(self):
+        self._set_current_disruption('NodetoolCleanupMonkey %s' % self.target_node)
+        cmd = 'nodetool -h localhost cleanup keyspace1'
```

Comment:
We might has multiple keyspaces, so we still only want to cleanup the first one?
All we will clean all of them one by one?